test/TestProjects/OperationGroupMappings/OperationGroupMappings.json

```diff
@@ -0,0 +1,325 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "OperationGroupsMapping",
+    "description": "OperationGroupsMappin Client.",
+    "version": "2020-06-01"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
```

Comment:
I recommend creating a minimal swagger required for testing this specific feature. So it's clear what's important for the scenario and what's not.

Reply:
https://dev.azure.com/azure-mgmt-ex/DotNET%20Management%20SDK/_workitems/edit/5729